test/use_case/login/LoginInteractorTest.java

```diff
@@ -0,0 +1,88 @@
+package use_case.login;
+
+import data_access.MemoryUserDataAccessObject;
+import entity.CommonUser;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LoginInteractorTest {
+    MemoryUserDataAccessObject userDataAccessObject;
+    String firstUser = "Bob";
+    String firstPassword = "pass";
+
+    @Before
+    public void setUp() throws Exception {
+        this.userDataAccessObject = new MemoryUserDataAccessObject();
```

Comment:
You can instantiate this above when you first declare this `userDataAccessObject`, so you don't have to reinstantiate it each time.